product-tours/tour-tf-keras/Tour-with-TensorFlow-and-Keras.ipynb

```diff
@@ -0,0 +1,2761 @@
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "eval_metrics = model.evaluate(x_test, y_test, verbose=0)\n",
+    "\n",
+    "for j, metric in enumerate(eval_metrics):\n",
+    "    neptune.log_metric('test_{}'.format(model.metrics_names[j]), metric)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "tests"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# tests\n",
+    "exp = neptune.get_experiment()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "UphOuRMm5sZs",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Step 7: Stop experiment at the end"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "This statement is needed only in Notebooks. In scripts experiment is closed automatically at the end."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "neptune.stop()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "tests"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# tests\n",
+    "## check logs\n",
+    "correct_logs = ['batch_loss', 'batch_accuracy', 'epoch_loss', 'epoch_accuracy', 'test_loss', 'test_accuracy',\n",
+    "                'epoch_val_loss', 'epoch_val_accuracy']\n",
+    "\n",
+    "if set(exp.get_logs().keys()) != set(correct_logs):\n",
+    "    raise ValueError('incorrect metrics')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "UphOuRMm5sZs",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "# More logging options"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "Below we present more logging options. Flow of the example is similar to the basic one. The difference is that here we present more logging options.\n",
+    "\n",
+    "* logging experiments parameters,\n",
+    "* logging data version,\n",
+    "* model architecture summary,\n",
+    "* weights of the trained model,\n",
+    "* predictions on the test set as table,\n",
+    "* visualizations of the model performance,\n",
+    "* training data sample."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Install additional dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "DO7yawFVxVIt",
+    "tags": [
+     "installation"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "! pip install --quiet scikit-plot==0.3.7 matplotlib==3.3.3"
```

Comment:
Shouldn't it be with matplotlib==3.2.0  to have them interactive?

Reply:
In this tour we do not use interactive. We discussed it earlir -> converged to soluton that you log img here, until we fix matploblib -> plotly issues.